app/services/description_generator.rb

```diff
@@ -159,5 +160,17 @@ def contributor(work_form_contributor)
       }
     end
   end
+
+  sig { returns(T::Array[T::Hash[String, T::Array[T::Hash[String, String]]]]) }
```

Comment:
ew.  but we probably have this in other spots.   this is where maybe we could use the references to cocina types?

Reply:
yeah, this looks yucky.  i'm about to push something that's... a little longer, but much clearer about the expected structure (imho), so hopefully that'll be less yuck?